TROUBLESHOOTING.md

```diff
@@ -0,0 +1,10 @@
+# Cluster Logging Operator Troubleshooting
+
+## Known issues
+
+### After Upgrading CLO and then EO, CLO doesn't know about `kibana` type
+Part of the work done for between CLO & EO 4.4 -> 4.5 was to change which operator was responsible for Kibana. This included creating a new CRD for Kibana that the EO watches for.
```

Comment:
The responsibility of deploying Kibana was moved to the EO from CLO as part of the 4.5 feature work. ....